cli/smartcontract/smart_contract.go

```diff
@@ -374,9 +374,10 @@ func initSmartContract(ctx *cli.Context) error {
 	}
 
 	gm := []byte("module " + contractName + `
-require (
+	go 1.18
```

Comment:
Fix the indent, please.

Reply:
Here and below. Run `neo-go smartcontract init` and see the generated file, it should have the correct indent.